src/api/db/migrate/20240710083919_add_type_column_to_notifications.rb

```diff
@@ -0,0 +1,5 @@
+class AddTypeColumnToNotifications < ActiveRecord::Migration[7.0]
+  def change
+    add_column :notifications, :type, :string
```

Comment:
maybe we should add an index to the column to improve performance

Reply:
What do you think?

Reply:
:+1: